Makefile

```diff
@@ -121,12 +121,13 @@ vet: gowork ## Run go vet against code.
 	go vet ./...
 	go vet ./... ./api/...
 
+APIPATH ?= $(shell pwd)/api
 .PHONY: tidy
 tidy: fmt
 	go mod tidy; \
-	pushd "$(LOCALBIN)/../api"; \
```

Comment:
Maybe this can be:

```
.PHONY: tidy
tidy: fmt
	go mod tidy
	cd ./api
	go mod tidy
```
and `APIPATH` is no longer required and can be deleted.

Reply:
@bshewale hi, can you update this patch with Francesco's suggestion?
I got a clean run with your patch + Francesco suggestion:
https://paste.opendev.org/show/buCwKc76sL73jFebBH6N/